src/tagmanager-wrap.js

```diff
@@ -0,0 +1,89 @@
+
+const defaultOptions = {
+  gtmId: '111111', // homolog
+  virtualPageViewEvent: 'virtual_pageview',
+  startPush: {
+    experiements: [],
+  },
+  addExperiments: [],
+};
+
+export default class TagManager {
+  constructor(dataLayer, params = defaultOptions) {
+    this.options = Object.assign({}, defaultOptions, params);
```

Comment:
We can remove this empty object `{}`.

Reply:
@felipeorlando The most common use is to have the first param as an empty object. That way  when you assign it with other object it creates a new object without changing the source obj

Reply:
@iondrimba is the smartest guy that I ever meet. He explain to me that the `Object.assign` modifies the first parameter.

Man, you is my mentor, a example to follow. The humanity needs you more than coffee, cars and money.

Reply:
@iondrimba @felipeorlando removing `{}`, we are assuming that defaultOptions will be merged (changed) with params.

In this case, I don't think that is a problem, but, as @iondrimba says, this is the most common use :)